### DISCUSSION TITLE
Spoof version

### DISCUSSION THREAD

**Discussion by alcatraz-zz** [kitadai31/revanced-patches-android6-7#3] [Q&A]
When using the recommended patches, what version is it now spoofed as? I'm curious because I'd like to have it working for as long as possible.

Thanks!

**Comment by kitadai31** [comment]
(Moved to discussions)

Version is not spoofed by default.
If you turn on "Enable new layout", it spoof app version to 18.05.40.

Also, applying "Force Enable New Layout" patch, it is always spoofed to 18.05.40 regardless of the setting.

**Comment by kitadai31** [comment]
By the way, is this post yours?

![image](https://user-images.githubusercontent.com/90122968/232667737-4aad7114-72a7-4b46-b2cb-a21e735304ca.png)

There is a misinformation in that post. Let me tell you about correct information.

You need the special version of Revanced Manager, **only if you want to run Manager on Android 7.x**.
If you intend to run Manager on newer device (Android 8.0+), you should use official ReVanced Manager instead of mine.

If you are able to edit that post, I would appreciate it if you could edit it.
(I'm not familiar with XDA forum so I don't know that post is able to be edited.)

**Comment by alcatraz-zz** [comment]
Fixed it! Cheers...

**Comment by kitadai31** [comment]
Thanks!
